qa/restart-upgrade/src/test/java/org/opensearch/knn/bwc/IndexingIT.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright OpenSearch Contributors
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.bwc;
+
+import static org.opensearch.knn.TestUtils.KNN_BWC_PREFIX;
+
+public class IndexingIT extends AbstractRestartUpgradeTestCase {
+    private final String CLASS_NAME = "indexingit";
+    private final String testIndexName = KNN_BWC_PREFIX + CLASS_NAME + "test-index";
```

Comment:
Why do you need class name to be part of index name?

Reply:
If same index name is being used in multiple test files for a given BWCTest(rolling upgrade or restart upgrade) it is throwing exception. So to make indices unique, I'm adding class name in index name.

Reply:
You should generate random index name and delete once it is not required. It will be problem if we don’t clean up resources that we create

Reply:
Now, we are deleting the index at the end of test.